client/src/modules/Job/pages/NoAuthLanding/components/DottedRectangle.tsx

```diff
@@ -0,0 +1,33 @@
+import { Space } from 'antd';
+import { COLORS } from '../../../theme/colors';
+
+export const DottedRectangle = () => {
+  const SIZE = new Array(40).fill(null);
+
+  return (
+    <Space size={32} wrap>
+      {SIZE.map((_, index) => (
+        <span key={index} className="dot"></span>
+      ))}
+
+      <style jsx>
```

Comment:
Not sure what you try to do here  but please consider to use corresponding antd component instead of tons of css